### PR TITLE
Initial implementation of feature layer geopackage sample

### PR DESCRIPTION
**Pull request by tschie**
feature layer geopackage sample

### CODE REVIEW

**Comment by tschie** [comment]
@john0005 Can you please review?

File: src/main/java/com/esri/samples/featurelayers/feature_layer_geopackage/FeatureLayerGeoPackageSample.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // create a map with the streets vector basemap
+      ArcGISMap map = new ArcGISMap(Basemap.createStreetsVector());
+
+      // create a GeoPackage from a local gpkg file
+      File geoPackageFile = new File("./samples-data/auroraCO/AuroraCO.gpkg");
```
Comment (john0005) [inline src/main/java/com/esri/samples/featurelayers/feature_layer_geopackage/FeatureLayerGeoPackageSample.java:60]: Didn't show up in samples data.

File: src/main/java/com/esri/samples/featurelayers/feature_layer_geopackage/FeatureLayerGeoPackageSample.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+          List<GeoPackageFeatureTable> featureTables = geoPackage.getGeoPackageFeatureTables();
+          if (featureTables.size() > 0) {
+            FeatureLayer featureLayer = new FeatureLayer(featureTables.get(0));
+            map.getOperationalLayers().add(featureLayer);
+            // zoom to the layer
+            featureLayer.addDoneLoadingListener(() ->
+              mapView.setViewpointAsync(new Viewpoint(featureLayer.getFullExtent()))
+            );
+          }
+        }
```
Comment (john0005) [inline src/main/java/com/esri/samples/featurelayers/feature_layer_geopackage/FeatureLayerGeoPackageSample.java]: Would add alert about geopackage not being found.

**Review by john0005 — CHANGES_REQUESTED** [review]

File: src/main/java/com/esri/samples/featurelayers/feature_layer_geopackage/FeatureLayerGeoPackageSample.java (hunk 1 above)
Comment (tschie) [reply src/main/java/com/esri/samples/featurelayers/feature_layer_geopackage/FeatureLayerGeoPackageSample.java:60]: Added it.

**Comment by tschie** [comment]
@john0005 I've made the changes. Please review again. Thanks.

**Review by john0005 — APPROVED** [review]

**Comment by john0005** [comment]
Looks good.

File: src/main/java/com/esri/samples/featurelayers/feature_layer_geopackage/FeatureLayerGeoPackageSample.java [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+      stage.setWidth(800);
+      stage.setHeight(700);
+      stage.setScene(scene);
+      stage.show();
+
+      // create a map with the streets vector basemap
+      ArcGISMap map = new ArcGISMap(Basemap.createStreetsVector());
+
+      // create a GeoPackage from a local gpkg file
+      File geoPackageFile = new File("./samples-data/auroraCO/AuroraCO.gpkg");
```
Comment (colinanderson) [inline src/main/java/com/esri/samples/featurelayers/feature_layer_geopackage/FeatureLayerGeoPackageSample.java:60]: A more modern way would be to do something like `Paths.get("./samples-data/auroraCO/AuroraCO.gpkg").toAbsolutePath()`.

**Pull request merged**